### ISSUE TITLE
10-7959c and 10-7959a: Spike - adding email to PDF

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#91232]
For the 2 forms that do not have email addresses on the PDF currently, we should investigate where the email address could go on the PDF.

**Comment by andrea-merrill** [comment]
@michaelclement is this needed? where do we stamp emails now for OHI and ChampVA Claims?

**Comment by michaelclement** [comment]
@andrea-merrill 
Right now we show emails on 10-10d (for applicants only), and FMP 1 + 2:
|10-10D|FMP1|FMP2|
|-|-|-|
|![Image](https://github.com/user-attachments/assets/0c245b96-5740-498f-9706-38ce6316289d)|![Image](https://github.com/user-attachments/assets/acff819a-a234-4933-94f6-fb77c6e4c37f)|![Image](https://github.com/user-attachments/assets/a7806184-a4d9-4c21-87dd-e789ed95e241)|

CHAMPVA claims and OHI don't have anywhere for emails to be displayed - I think before we do any work on this we should check with Premal and also look at what the 2027 versions of these forms look like (if they've been released). It might not make sense to be slapping extra info on the form if it's not actually requested.

|Claims|OHI|
|-|-|
|![Image](https://github.com/user-attachments/assets/3c376def-71d0-4ea4-8489-abf3651ecc49)|![Image](https://github.com/user-attachments/assets/6ecfbcab-3a09-498c-97e9-cfa1108f3d41)|

**Comment by andrea-merrill** [comment]
and we send the signer email in the metadata for ChampVA claims and OHI?

**Comment by michaelclement** [comment]
Yep, accessible to Cindy/DOCMP via the metadata property `primary_contact_email`
